_projects/information-architecture/web-app.md

```diff
@@ -3,3 +3,87 @@ title: Web App
+
+
+## Setup
+
+- Individual
+- 6 class days <small>(plus the weekend!)</small>
+- Standups every morning in the week leading up to the official start date of the project
+
+
+## Materials
+
+- Sketch
+- Invision
+- Principle
+
+
+## Project Scope
+
+You will be designing a new solution. If the company is already using a product, then it is a competitor, and you should design a superior software. Don’t try to change the existing product. 
+
+Because you are launching this product, you need to ensure that this is a service that would be useful to multiple companies. 
+
+
+## Deliverables
+
+- Prototype of the MVP for a desktop (hi-fidelity)
+- Roadmap for next 3-6 months (additional features beyond the MVP that will be rolled out in the future)
+- Landing page that explains what your product is (optional) 
+- Presentation of research and final proposal
+- Report 
+
+
+## Research
+
+You are required to find people inside companies to talk to. You should focus on startup companies in the tech industry that have grown to a number where they are starting to need some HR-related system (at least 8-10 people). This is why it is critical to start ahead so that you have time to plan and visit offices prior to the dedicated project week. Research can be done through Skype if you are connected to companies elsewhere.
+
+
+## Suggested categories <small>Pick only one</small>
+
+- Vacation / Time-off Management
+- Performance Reviews 
+- Time Tracking 
+- Benefits Management 
+- Hiring Process 
+- Filing Complaints Internally
+- Onboarding Process 
+- Professional Development 
+
+Considerations to assist in determining where to focus:
+- Is this area functional? = Is the system already in place and working well? If it is, rule out and move on
+- Is this area dysfunctional? = The system is in place but lots of complaints, not well executed, working poorly. How can we make this better and compete in the space? It is a good area to explore
+- Is this area ad Hoc (thrown together) or non-existent? = Not yet a complete system or process, nothing concrete in place. There is a lot of room for innovation and ideation and it is a good area to explore. 
+
+## Suggestions
+
+- Find someone you want to talk to and in your initial conversation investigate the above and narrow down the pain points to determine in which area to focus on. This is generative research. 
+- Expand your research to other companies based on the area you have decided to focus on
+- Find someone who works in HR to get their perspective, as well as employees (preferably low-level).
+- Look to LinkedIn, Slack or leverage your existing networks to find potential contacts and interviewees
+- Run usability tests with real users in coworking spaces (WeWork, Spaces)
+- Early in the week try to visit companies and shadow employees to observe their current process, behaviors and pain points
+
+## Presentation (6 minutes)
+
+Your presentation should answer the following questions:
+
+1. 
```

Comment:
What did your research show you?
What variations on your design did you explore?
How were your design decisions connected to your research findings?
How would a user operate your final design?
How did your design change based on testing with users?
What features are on your 3–6 month roadmap?
How did you determine which features to ship first?